src/style.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, Helvetica, sans-serif;
+  margin-top: 0;
+  box-sizing: border-box;
+}
+
+#gsearch {
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+}
+
+#gsearch1 {
+  width: 100%;
+  height: 42px;
+  font-weight: 300;
+  font-size: 14px;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
```

Comment:
you repeat a lot of styles in these two classes go through [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) and look closely at 2nd item